### ISSUE TITLE
Chat color picker or set wallpaper picker aren't accessible via VoiceOver

### ISSUE THREAD

**Issue by atfelix** [signalapp/Signal-iOS#5662]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description

The wallpaper presets are not accessible via VoiceOver

### Steps to reproduce

- Enable VoiceOver
- Navigate to the set wallpaper screen or chat color screen
- Attempt to navigate past the presets heading (set wallpaper screen) or the color picker (chat color screen)

**Actual result:** You cannot navigate past the presets or inside the color picker

**Expected result:** You can select a preset for your wallpaper or select a color

### Device info

**Device**: iPhone X

**iOS version**: iOS 16.6

**Signal version:** 6.37.0.8

**Comment by Miguel-Signal** [comment]
Hi @atfelix 

I've gone ahead and filed this bug with our engineering team. While our engineers will be working on a fix, I unfortunately can't give out a specific timeline for a resolution. You may see a fix for this bug included in the release notes for future versions of Signal.

Thanks again for your help in reporting this!

**Comment by marissa-signal** [comment]
Hi - This issue will be fixed in version 6.54, which goes to beta next week.
